artemis-commons/src/main/java/org/apache/activemq/artemis/utils/collections/TypedProperties.java

```diff
@@ -174,10 +174,10 @@ public Boolean getBooleanProperty(final SimpleString key) throws ActiveMQPropert
       Object value = doGetProperty(key);
       if (value == null) {
          return Boolean.valueOf(null);
-      } else if (value instanceof Boolean) {
-         return (Boolean) value;
-      } else if (value instanceof SimpleString) {
-         return Boolean.valueOf(((SimpleString) value).toString());
+      } else if (value instanceof Boolean boolean1) {
+         return boolean1;
+      } else if (value instanceof SimpleString string) {
+         return Boolean.valueOf(string.toString());
```

Comment:
Things like boolean1 (plus byte1, short1, float1,and double1 etc that follow below) are more than a bit ugly, probably even less readable compared to the casts.

Calling the SimpleString _string_ results in the silly reading `string.toString()`.

Reply:
Fixed.